### PR TITLE
Szgula/enginfra 752 add nix support

### PR DESCRIPTION
**Pull request by szgula**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "builddsl"
-version = "1.0.0"
+version = "1.0.1"
```
Comment (NiklasRosenstein) [inline pyproject.toml]: The bump happens when I run `slap release` :)

```suggestion
version = "1.0.0"
```

File: .changelog/1.0.1.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+release-date = "2023-04-03"
```
Comment (NiklasRosenstein) [inline .changelog/1.0.1.toml]:
```suggestion
```

Same here, and that would go into `.changelog/_unreleased.toml` until I run `slap release` after merge.

File: pyproject.toml (hunk 1 above)
Comment (szgula) [reply pyproject.toml]: ups, sorry - I did `slap release` on my fork repo =D ...I'll revert it now

File: pyproject.toml [hunk 3]
```diff
@@ -20,9 +20,13 @@ python = "^3.6"
...
 nr-io-lexer = "^1.0.0"
 dataclasses = { version = "^0.6", python = "<3.7" }
+astor = { version = "^0.8.1", optional = true }
 
 [tool.poetry.extras]
-astor = ['astor ^0.8.1']
+astor = ["astor"]
+
+[tool.poetry.scripts]
+builddsk = "builddsl.wrapper.__main__:main"
```
Comment (NiklasRosenstein) [inline pyproject.toml]: Where.. does this come from? `builddsl.wrapper` does not exist

Comment (szgula) [reply pyproject.toml]: this is actually not needed

**Review by NiklasRosenstein — APPROVED** [review]

**Pull request merged**